### ISSUE TITLE
Deploy to the web?

### ISSUE THREAD

**Issue by scottrblock** [ultorg/public_issues#24]
Hi, I'm currently using the macos build of the beta with a fairly large sqlite db, and its working great. I'd like to give others a way to to access this db and ultorg on top of it, so I was wondering if I could somehow build ultorg to something I could deploy to the web. Mostly would just like to learn if this is something on the team's radar/roadmap, and what that future might look like!

**Comment by eirikbakke** [comment]
Hi, @scottrblock! To collect a datapoint here, roughly how large is the SQLite file in question? 1MB, 10MB, 100MB, 1GB? Does this database ever get updated? Do individual users need to update it?

I'm working on a user accounts system for Ultorg now, initially for billing, but later to help support collaborative functionality. For example, I'm planning to let users share perspectives (queries) over external data sources, and eventually also to let Ultorg host tables of data. The vision is to let users share data like in Google Sheets, but from the full-fledged desktop app.

For more occasional users, there should eventually be a way to share a link to a perspective which can be opened in a web browser. But this would be a very simplified page, which should load quickly and not require much of a learning curve. Input is welcome!

On the current version of Ultorg, you can always dump your data into Google Sheets (if small enough) and connect to it there. But each user would need to download the desktop app. (Do feel free to encourage colleagues to fill in the [signup form](https://www.ultorg.com/signup)! :-)

**Comment by scottrblock** [comment]
thanks @eirikbakke. the db is currently 3.1gb, but I could likely get that down to ~1gb if needed. User accounts sounds great, would there eventually be an API to manage a team of users?

**Comment by eirikbakke** [comment]
Does the data in the database change, or is it a static dataset of some sort?

APIs are a bit into the future; I'd need to hear how various potential customers would like to use them. Ideally I'd like to keep the product "no-code", i.e. avoid the need to do any kind of custom coding.

**Comment by scottrblock** [comment]
Good question! It is currently static, but would be most valuable as something that could update a few times a day. Still would be valuable to folks in its current static form though! hope that makes sense 😆

**Comment by eirikbakke** [comment]
Makes sense! It's a fairly large database; too big to do tricks like just giving each user a complete new copy on every update. I don't know the details, but if you have many users, it sounds like it would need to end up in something else than SQLite. Hopefully one day Ultorg can host databases like this -)

**Comment by MarkRudChristensen** [comment]
Hi. I am looking very much forward to these collaborative features. My need currently is to share the perspectives I have made as part of demoing the tool to my colleges. Until this kind of collaboration is featured I was thinking about simply sending the perspective save files manually, but I cannot find them. 

Can you @eirikbakke tell me where they are stored?

**Comment by eirikbakke** [comment]
Hi, @MarkRudChristensen! At the moment there isn't a way to export perspectives, but this is obviously needed. I've added issue https://github.com/ultorg/public_issues/issues/32 for this.

(Ultorg stores its data in the OS-specific "Application Data" folder, in a PostgreSQL database.)
